bin/script.js

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env node
```

Comment:
What is `script.js` supposed to do?

Reply:
It was just used to put the new `profileIDs` field in the indexes with all the information. I removed it since it was only for single use.